test/quantization/core/test_workflow_module.py

```diff
@@ -327,6 +327,35 @@ def test_histogram_observer_consistent_buffer_shape(self):
         self.assertEqual(min_shape_before, obs.min_val.shape)
         self.assertEqual(max_shape_before, obs.max_val.shape)
 
+    def test_histogram_observer_ignore_infinity(self):
+        """
+        Ensures that HistogramObserver doesn't record values of infinity
+        """
+        obs = HistogramObserver()
+        obs2 = HistogramObserver()
+        x = torch.randn(4, 4, 4, 4)
+        obs(x * torch.inf)
+        obs(x)
+        obs2(x)
+        obs(x * torch.inf)
```

Comment:
Is this meant to check negative infinity?

Reply:
yeah it checks both positive and negative infinity, basically running obs(x*torch.inf) shouldn't do anything.